skrf/tests/test_network.py

```diff
@@ -125,8 +126,23 @@ def test_time_transform(self):
     def test_time_transform_multiport(self):
         dut_dc = self.ntwk1.extrapolate_to_dc()
 
-        with self.assertRaises(ValueError):
-            t, y = dut_dc.step_response()
+        y1 = npy.zeros((1000, dut_dc.nports, dut_dc.nports))
+
+        for (i,j) in dut_dc.port_tuples:
+            oneport = getattr(dut_dc, 's{0}{1}'.format(i+1, j+1))
```

Comment:
```suggestion
            oneport = getattr(dut_dc, f's{i+1}{j+1}')
```
Minor suggestion for code readability, as we now support Python >= 3.6.